internal/dag/builder_test.go

```diff
@@ -6271,8 +6271,8 @@ func TestDAGInsert(t *testing.T) {
 					Prefix: "/",
 				}},
 				Services: []contour_api_v1.Service{{
-					Name: "kuard",
-					Port: 8080,
+					Name: "nginx",
```

Comment:
Why swap out for `nginx`?

Reply:
I think this has also been wrong all the time: The proxy is used in test _"insert proxy with response header policy - route - host header"_ [here](https://github.com/projectcontour/contour/blob/0a94c00f2bf748e1afee914c8ce5fe06b0707a60/internal/dag/builder_test.go#L8913) and it is expected to return no listeners. However, the reason why it returned no listeners previously was because the proxy has invalid service reference. The test includes service `s9` which is `nginx`, not `kuard`. Now that invalid service reference will result in listener, I catched this error.